SPECS/edk2/edk2.spec

```diff
@@ -115,6 +115,11 @@ Patch1001: CVE-2023-3817.patch
 Patch1002: CVE-2023-0465.patch
 Patch1003: CVE-2023-2650.patch
 Patch1004: improve-safety-of-DH.patch
+Patch1005: CVE-2022-36763.patch
+Patch1006: CVE-2022-36764.patch
+Patch1007: CVE-2022-36765.patch
+Patch1008: CVE-2023-45230.patch
+Patch1009: CVE-2023-45236.patch
```

Comment:
Please also update the `%prep` section to apply the patches. It looks like the patches starting from number 1000 and up are reserved for changes in the `CryptoPkg/Library/OpensslLib/openssl` directory. You may need to change the patch number, if these fixes are to be applied directly to the main source folder or tweak the steps in the `%prep` section.

Reply:
Good catching. This is resolved